package.json

```diff
@@ -257,7 +257,6 @@
         "@types/del": "^3.0.1",
         "@types/fs-extra": "^5.0.4",
         "@types/glob": "^7.1.1",
-        "@types/handlebars": "^4.0.39",
```

Comment:
`handlebars` 4.1.0 contains type definitions, so we don't need to depend on `@types/handlerbars` (which requires us to keep its version in-sync with `handlebars`).